### DISCUSSION TITLE
Files missing on Github Pages

### DISCUSSION THREAD

**Discussion by Pate35** [community/community#46423] [New to GitHub]

**Comment by shayan1234554321** [comment]
If certain files, such as CSS files or the "javax.faces.resource" folder, are not being displayed on your GitHub Pages website, it could be because of a few different reasons 😭 :

1. File name casing ✍️ : GitHub is case-sensitive, so make sure that the filenames in your repository match the filenames in your HTML code exactly.
2. File location 🚴‍♀️ : Ensure that the CSS and resource files are stored in the correct directory in your repository and that the paths in your HTML code are correct.
3. Custom domain 🏅 : If you're using a custom domain for your GitHub Pages website, make sure that it's properly set up and that the custom domain is listed in your repository's CNAME file.
4. Accessibility 🧩 : If you've set up a private repository for your website, make sure that the files are accessible to the public. If you're using a public repository, the files should be accessible by default.

If none of these steps resolve the issue, you can try checking the repository's settings and the GitHub Pages documentation for further information 😀 .

**Comment by Ntshangase** [comment]
You can checkout [https://docs.github.com/en/pages](url) for more information. Or log a ticket if the documentation doesn't assist you after trying it's solution out.

**Comment by lord-xld3** [comment]
This might help someone else. Given a tree like this
|- index.html
|- /img
    |- something.png

Your img in html needs to be img/something.png, NOT /img/something.png

**Comment by marczis** [comment]
I get the same issue. Other pictures are working. If I rename the file and push again it starts to work. 
Strange - I sense a bug, but can't really investigate further.
